### DISCUSSION TITLE
Extra Credit Visiting Voice: Rapi Castillo

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#23] [Extra Credit]
### Due 8/2

View the Extra Visiting Voice and respond in 3-4 sentences about how you believe activism and technology are connected.

**Comment by cynthinee** [comment]
I really like how this talk repeatedly went back to the issue of engineers and tech workers needing to have relationships and hear the stories of people in the communities that they're attempting to provide solutions for. I'm not sure how we got here, but it seems like most industries are totally okay with the 'experts' at the top providing solutions based on their assumptions of problems with no direct input from actual people. And I think as technologies have developed, this is also where technological determinism comes into play as also being THE solution to problems. Expertise and technology can be great but not without the aforementioned relationships and stories, as well as ethics.

I think activism is an intervention to this unbalanced power dynamic that attempts to rebalance things: experts, communities, ethics. And technology can have a place in this rebalancing, as evidenced by Rapi's work. 

Right now with the pandemic, an issue that I'm personally invested in is reopening of college campuses and their libraries. In this situation, a lot of administrators who aren't librarians and/or aren't the ones who have to be working directly with the public are making decisions about reopening, which is kind of terrifying. Library workers, myself included, share our struggles with terrible reopening plans and experiences with the hashtag #ProtectLibraryWorkers, and I was just thinking that if I didn't have to write my dissertation proposal right now, I could probably take the skills learned in this class and map these stories in some way. To the question about what our passions are, I think my passion lies in being an activist for the communities I'm a part of and partnering with other communities to help in collective struggles. 

This visiting voice also made me really appreciate Reddit. The [r/SandersforPresident](https://www.reddit.com/r/SandersForPresident/) subreddit mobilized organizers in which Rapi was able to identify a need of the community to easily find events geographically and aggregate events on a map while learning Leaflet. Several students found this course via the UCLA subreddit. One of my favorite things about the Reddit platform is that it hasn't transformed significantly since it was created ([example random snapshot from 2007](https://web.archive.org/web/20071002144324/http://reddit.com/)). A lot of social media platforms are always developing the technology to 'keep up' with other platforms and then the original reason that platform was created loses utility. I think Reddit is a great example of a platform that sticks to what it's good at, which is also what users want. It's consistency also makes it reliable to users, in my opinion, and perhaps that's a reason why it has the capacity to help bring people together. (A lot more to say about that since obviously the bringing people together part has also contributed to some really bad things.)

Lastly, I will just plug the optional lecture–highly recommend!

**Comment by abi-200** [comment]
Similar to how we discussed in lecture before, justice and activism can use technology as a tool to enhance certain voices and projects, but it is not the end-all solution for any social issue. This talk was very powerful to listen to because it really highlighted the need for community involvement in true activism, and how far too often, people affected by social change plans are not the ones creating the plans due to systemic policies and institutions. 

Many of my peers, especially at UCLA, have said their end goal is to be a leader or person at the top of a command chain. This is not inherently a bad thing; as Rapi said, the collective talent and potential of both ourselves and our peers can be a powerful tool, but only if we use this privilege in the right way, and collect viewpoints and voices ethically and responsibly. In certain career fields, such as in tech, the climb for more money and to the top is obviously very enticing; digital media and tech is a very lucrative field. This talk made me appreciate how our lectures had a core focus on ethics and justice and did not treat this as a side subject of technology; technology without ethical consideration has high potential for harm. 

The Amazon Headquarters example was also a beautiful example of how technology can yield a disproportionate amount of  power, but it does not have to mean the overpowering of communities. Too often, immigrant communities are viewed as an easier target of which to take advantage. I’m not sure to what degree this is due to prejudice/income/perceived lack of political power, but regardless, gentrification is a major problem that acts in detriment to many families and people in such communities. The list of unions and coalitions that helped with this movement, all from a diverse set of background that do not normally have a lot of power individually, coming together to work towards a common goal really moved me and is a testament to the power of community voices.The tech workers in these coalitions shows an example of the other side of the coin mentioned above: tech workers do not have to use their skills to empower corporations, and technology can and does have a meaningful place in activism when needed.

I’ve been active on Reddit for a long time, and it’s my preferred platform, but I never really heard too much about how it can be used for social projects and change in the way that it was been highlighted in this video. I’ve heard of Facebook campaigns and events as mentioned, and I follow some subreddits dedicated to social causes, but the net impact of such subreddits and the projects that could be created was something new for me to hear. I enjoyed hearing about how Rapi was able to use open-source technologies to create a map that could later be used in campaigns - I think this is a wonderful example of how technology can be a great tool used to advance social change, but is just that - a tool.

**Comment by ronnie-blanche** [comment]
A point I really appreciated about this conversation was how Rapi drives home that people creating technology, like engineers, should also seek a friendship/ties to the communities they’re trying to aid — similar to Yoh! Technology can only be used at its highest potential when the people creating it truly understand and recognize the struggles and obstacles the communities they’re assisting face. If there is no understanding, then the product is just created to help uphold capitalism. The issue here is that capitalism is a key force in preserving the oppressive institutions we already have in place. Capitalism helps to advance sexism, classism, racism, and colonialism — all of which propel white supremacy and future oppression of marginalized groups. So while technology can be used as a platform to spread and uphold activism, it can also be the very undoing of these efforts if technology is created for the wrong reason/lacks the ethics that we very much value in this class.

**Comment by eandrusenki** [comment]
Because of the day and age we are in, I truly believe that technology has become an important part of activism, (not THE most important but), especially because as Abita ,mentioned, and we have learned a lot about in this class, we can use technology as a way to empower people within a community and pass on our knowledge to them. This can allow them to educate people beyond with technology.  Something I find interesting and I really like is where Rapi says in his presentation "we can either help corporations or uplift and empower the people" because I think it is important that we also promote autonomy and allow people to have the choice, and choose the "right" thing, or the path of empowering communities working on activism. 
As Rapi mentioned, it is also important that members of the tech community need to have relationships with people in other communities. Relationships allow for an aspect of humanity, compassion and feelings that we can't get with technology or the people at the top who are in power.

**Comment by ramirezirvin** [comment]
As noticed the younger generations use social media platforms to spread awareness over social justice issues. Some partner up with campaigns or repost and inform other individuals what is going on in the world, aside from news sources a lot of people spread this information through twitter, instagram, reddit and other social media.  If technology is used correctly, it can spread its intended message to uplift individuals and share spotlights to spread awareness of important events or causes, to empower the voices that are not heard. If people are more informed and aware of what is happening around them they are willing to spread more awareness of over the knowledge they obtained and participate in changing the world one step at a time.
The relationships, created with individuals branching out to multiple communities, as Rapi mentioned (such as between tech/engineer workers and community members) can help us be more human and understand one another, and as many people consume everything through technology nowadays it could facilitate this gap we have in society.
